pkgs/applications/networking/instant-messengers/fractal/default.nix

```diff
@@ -1,102 +1,84 @@
 { stdenv
 , lib
 , fetchFromGitLab
-, nix-update-script
 , cargo
 , meson
 , ninja
-, gettext
-, python3
 , rustPlatform
 , rustc
 , pkg-config
-, gtksourceview4
 , glib
-, libhandy_0
-, gtk3
-, dbus
+, gtk4
+, gtksourceview5
+, libadwaita
+, gstreamer
+, gst-plugins-base
+, gst-plugins-bad
+, desktop-file-utils
+, appstream-glib
 , openssl
+, pipewire
+, libshumate
+, wrapGAppsHook4
 , sqlite
-, gst_all_1
-, cairo
-, gdk-pixbuf
-, gspell
-, wrapGAppsHook
+, xdg-desktop-portal
 }:
 
 stdenv.mkDerivation rec {
   pname = "fractal";
-  version = "4.4.2";
+  version = "5";
 
   src = fetchFromGitLab {
     domain = "gitlab.gnome.org";
     owner = "GNOME";
     repo = "fractal";
     rev = version;
-    hash = "sha256-/vPadtyiYDX0PdneMxc0oSWb5OYnikevqajl3WgZiGA=";
+    hash = "sha256-XHb8HjQ5PDL2sen6qUivDllvYEhKnp1vQynD2Lksi30=";
   };
 
   cargoDeps = rustPlatform.importCargoLock {
     lockFile = ./Cargo.lock;
     outputHashes = {
-      "either-1.5.99" = "sha256-Lmv9OPZKEb7tmkN+7Mua2nx0xmZwm3d1W623UKUlPeg=";
-      "gettext-rs-0.4.2" = "sha256-wyZ1bf0oFcQo8gEi2GEalRUoKMoJYHysu79qcfjd4Ng=";
-      "sourceview4-0.2.0" = "sha256-RuCg05/qjkPri1QUd5acsGVqJtGvM5OO8/R+Nibxoa4=";
+      "matrix-sdk-0.6.2" = "sha256-X+4077rlaE8zjXHXPUfiYwa/+Bg0KTFrcsAg7yCz4ug=";
+      "mas-http-0.5.0-rc.2" = "sha256-XH+I5URcbkSY4NDwfOFhIjb+/swuGz6n9hKufziPgoY=";
     };
   };
 
   nativeBuildInputs = [
-    gettext
+    glib
+    gtk4
     meson
     ninja
     pkg-config
-    python3
-    cargo
+    rustPlatform.bindgenHook
     rustPlatform.cargoSetupHook
+    cargo
     rustc
-    wrapGAppsHook
-    glib
+    desktop-file-utils
+    appstream-glib
+    wrapGAppsHook4
   ];
 
   buildInputs = [
-    cairo
-    dbus
-    gdk-pixbuf
     glib
-    gspell
-    gst_all_1.gst-editing-services
-    gst_all_1.gst-plugins-bad
-    gst_all_1.gst-plugins-base
-    (gst_all_1.gst-plugins-good.override {
-      gtkSupport = true;
-    })
-    gst_all_1.gstreamer
-    gst_all_1.gst-devtools
-    gtk3
-    gtksourceview4
-    libhandy_0
+    gstreamer
+    gst-plugins-base
+    gst-plugins-bad
+    gtk4
+    gtksourceview5
+    libadwaita
     openssl
+    pipewire
+    libshumate
     sqlite
+    xdg-desktop-portal
   ];
 
-  postPatch = ''
-    chmod +x scripts/test.sh
-    patchShebangs scripts/meson_post_install.py scripts/test.sh
-  '';
-
-  preConfigure = ''
-    export GETTEXT_DIR="${gettext}"
-  '';
-
-  passthru = {
-    updateScript = nix-update-script { };
-  };
-
   meta = with lib; {
     description = "Matrix group messaging app";
     homepage = "https://gitlab.gnome.org/GNOME/fractal";
-    license = licenses.gpl3;
-    maintainers = teams.gnome.members ++ (with maintainers; [ dtzWill ]);
-    platforms = platforms.unix;
```

Comment:
Why was `platforms` dropped?

Also, I've requested review from both the maintainers, but I would be inclined to merge them into the new release rather than replacing.

Reply:
I don't know. It's dropped in fractal-next in https://github.com/NixOS/nixpkgs/pull/182862, not by me.

> I would be inclined to merge them into the new release rather than replacing.

What do you mean?

Reply:
Looks like you got what I meant.